dot-studio/golang

```diff
@@ -137,14 +170,14 @@ function install_go_tool() {
 document "go_build" <<DOC
   Builds the $pkg_name binary. (a.k.a. go build)
 
-  @(arg:1) The path to the go main package (default: cmd/${pkg_name}/${pkg_name}.go)
+  @(arg:1) The path to the go main package (default: main.go)
```

Comment:
This change is mainly because of the pattern `cmd/pkg/pkg.go` that lives only in Automate but it
is not being used there so, I would propose to default to any repo that has a `main.go`, things
like https://github.com/chef/chef-analyze